CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - Support for the font Gregoria.
 
 ## [Unreleased][ureleased]
+### Changed
+- `gregoriosyms.sty` is no more, its functionalities have been integrated in GregorioTeX.
+
```

Comment:
This should be moved to the Unreleased section above.  This Unreleased section is for 3.0.0.